src/content/app/tools/blast/utils/tests/input-sequence-examples/007.txt

```diff
@@ -1,4 +1,4 @@
 		  1 qikdllvsss tdldttlvlv naiyfkgmwk tafnaedtre mpfhvtkqes kpvqmmcmnn
 		 61 sfnvatlpae kmkilelpfa sgdlsmlvll pdevsdleri ektinfeklt ewtnpntmek
 		121 rrvkvylpqm kieekynlts vlmalgmtdl fipsanltgi ssaeslkisq avhgafmels
-		181 edgiemagst gviedikhsp eseqfradhp flflikhnpt ntivyfgryw sp
+		181 edgiemagst gviedikhsp eseqfradhp flflikhnpt ntivyfgryw sp$!
```

Comment:
Umm, let's not add the dollar and an exclamation mart to this example. It's a realistic input; and the dollar with an exclamation mark make it contrived. If we want to test the exotic nonsensical characters, let's have a separate test for that.